Tasks/google-play-release/GooglePlay.ts

```diff
@@ -768,30 +773,51 @@ function resolveGlobPaths(path: string): string[] {
     return [];
 }
 
+function getObbFromParentDirectroy(apkPath: string): string {
+    let obbFile: string = null;
+    let apkDirectory = path.dirname(apkPath);
+    let parentDirectory = path.dirname(apkDirectory);
+    let filenames = fs.readdirSync(parentDirectory);
+    filenames.forEach(file => {
+        if (path.extname(file) === '.obb') {
```

Comment:
If there are multiple `.obb` files what should happen?  This code will pick the last of them.  Why not the first?  If it were the first, there's no need to loop through the rest of them.  In any case, I think that `forEach` is not really what you want linguistically.  This should be a `for( const file of filenames)` loop.  Or a `filenames.find(file => path.extname(file) === '.obb')`

Reply:
Also, how is case sensitivity handled here?  (To be honest, I think this extension does not handle case very well, so perhaps that should be handled separate from this PR.)

Reply:
This is also wrong.  You want this:
```
const obbFile: string | undefined = filenames.find(file => path.extname(file) === '.obb');
```